allennlp/models/coreference_resolution/coref.py

```diff
@@ -78,12 +78,11 @@ def __init__(
 
         self._text_field_embedder = text_field_embedder
         self._context_layer = context_layer
-        self._antecedent_feedforward = TimeDistributed(antecedent_feedforward)
-        feedforward_scorer = torch.nn.Sequential(
-            TimeDistributed(mention_feedforward),
-            TimeDistributed(torch.nn.Linear(mention_feedforward.get_output_dim(), 1)),
+        self._mention_feedforward = TimeDistributed(mention_feedforward)
+        self._mention_scorer = TimeDistributed(
+            torch.nn.Linear(mention_feedforward.get_output_dim(), 1)
         )
-        self._mention_pruner = Pruner(feedforward_scorer)
```

Comment:
I think this is going to break the serialised coref models, because it will have the parameters in a different place now?

Reply:
Yes. I already have a model trained with `maked_topk`, and anticipate to get a final model ~tomorrow. I will update the model then.